mod/metrics/lib/javascript/script_metrics.js.coffee

```diff
@@ -19,23 +19,46 @@ decko.slotReady (slot) ->
   $('body').on 'change', ".TYPE-metric .card-editor.RIGHT-hybrid input[type=\'checkbox\']", (event) ->
     showResearchAttributes($(event.target))
 
+  $('td.metric-weight input').on 'keyup', (event) ->
+    activeEqualize()
+
+  $('#equalizer').on 'click', (event) -> 
+    if $(this).prop('checked') == true 
+      toEqualize( $('.wikiRating-editor') )
+
 showResearchAttributes = (checkbox) ->
   form = checkbox.closest("form")
+
   show_or_hide = checkbox.prop "checked"
   $.each ["value_type", "research_policy", "report_type", "methodology"], (_i, key) ->
     form.find(".card-editor.RIGHT-" + key).toggle show_or_hide
-
+    
 
 decko.editorContentFunctionMap['.pairs-editor'] = ->
   JSON.stringify pairsEditorHash(this)
 
 pairsEditorHash = (table) ->
   hash = {}
-  table.find("tbody tr").each ->
+  variableMetricRows(table).each ->
     cols = $(this).find('td')
     if (key = $(cols[0]).data('key'))
       hash[key] = $(cols[1]).find('input').val()
-  hash
+  hash 
+
+getValuesFromTable = (table) -> 
+  values = []
+  variableMetricRows(table).each -> 
+    tr = $(this) 
+    values.push( tr.find('td.metric-weight').find('input').val() )
+  values = values.splice(0, values.length - 1);
+  return variableValuesAreEqual(values)
+
+ # if all values are equals return "true"
+ variableValuesAreEqual = (values) ->
```

Comment:
I think this can be shorter, just:
```
values.every( (val, i, arr) => val == arr[0] ) == true
```
no?  why is the aux needed/helpful?

I would like to see Jest tests for this.